File: app/src/main/java/com/nitronapps/brsc_diary/DayModel.java
package com.nitronapps.brsc_diary;

public class DayModel {
    private int count;
    private String[] lessons, homeworks, marks;
    private boolean isWeekend;

    DayModel(int x, String[] y, String[] z, String[] a, boolean b){
        count = x;
        lessons = y;
        homeworks = z;
        marks = a;
        isWeekend = b;
    }

    public int getCount() {
        return count;
    }

    public String[] getHomeworks() {
        return homeworks;
    }

    public String[] getLessons() {
        return lessons;
    }

    public String[] getMarks() {
        return marks;
    }

    public boolean isWeekend() {
        return isWeekend;
    }
}

File: app/src/main/java/com/nitronapps/brsc_diary/Models/DayModel.kt
package com.nitronapps.brsc_diary.Models

import com.google.gson.annotations.SerializedName

class DayModel internal constructor(@SerializedName("count") val count: Int,
                                    @SerializedName("lessons") val lessons: Array<String>,
                                    @SerializedName("homeworks") val homeworks: Array<String>,
                                    @SerializedName("marks") val marks: Array<String>,
                                    @SerializedName("isWeekend") val isWeekend: Boolean,
                                    @SerializedName("dayName") val dayName: String,
                                    @SerializedName("teacherComment") val teacherComment: Array<String?>,
                                    @SerializedName("hrefHw") val hrefHw: Array<Array<String>?>?,
                                    @SerializedName("hrefHwNames") val hrefHwNames: Array<Array<String>?>?){
    fun printModel(): String{
        var result = ""

        result += dayName + "\n"
        result += count.toString() + "\n"

        return result
    }
}
